project-koku/nise
# Cost 2380 OCI generators

**This PR is part of the work to resolve [COST-2380](https://issues.redhat.com/browse/COST-2380)**

- create oci generator abstract class
- add oci compute generator
- add oci network generator
- add unit tests for the respective generators
- initial work on block storage generator


**Testing Instructions:**

- Run this command:
   - make test test_source=oci

- Expect to see:
   - number of tests run and the time taken in seconds,  e.g `Ran 66 tests in <##>s`
